articles/virtual-machines/workloads/redhat/redhat-rhui.md

```diff
@@ -211,13 +212,14 @@ australiaeast - 20.248.180.252
 southeastasia - 20.24.186.80
 
 # Azure US Government.
-# To be deprecated after 10th April 2023.
+# The below mentioned IPs are deprecated since 10th April 2023. Newer RHEL images are already redirected to Public region for updates. If you have already added below IPs to your UDR/firewall, you are not required to remove these IPs until next update on this doc.
 # For RHUI 4 connections, use public RHUI IPs as provided above.
 13.72.186.193
 13.72.14.155
 52.244.249.194
-
 ```
+> [!NOTE]
+> As of October 12th 2023 all PAYG clients will be directed to the RHUI4 IPs in phases over next two months. During this time the RHUI3 IPs will exist for continued updates but will be removed at a future time. Existing routes and rules allowing access to RHUI3 IPs should be updated to also include RHUI4 IP addresses for uninterrupted access to packages and updates. You should not remove RHUI3 IPs to continue receiving updates during the transition period.
```

Comment:
nit:
As of October 12th, 2023 all PAYG clients will be directed to the RHUI4 IPs in phases over next two months. During this time, the RHUI3
(added 2 commas)

Reply:
modified more, please check the new version

Reply:
![image](https://github.com/MicrosoftDocs/azure-docs/assets/50854544/5e64e372-7fba-46a6-b255-8532260c9ad3)